protos/engine/remote_engine.proto

```diff
@@ -5,12 +5,14 @@ package remote_engine;
 
 import "engine/schema.proto";
 import "engine/time_range.proto";
+import "cluster.proto";
 import "common.proto";
 import "storage.proto";
 
 service RemoteEngineService {
   rpc Read(ReadRequest) returns (stream ReadResponse) {}
   rpc Write(WriteRequest) returns (WriteResponse) {}
+  rpc GetTableInfo(GetTableInfoRequest) returns (GetTableInfoResponse) {}
```

Comment:
What is this used for?

Reply:
The schema of partition table will be obtained from the first sub-table, so a remote engine is needed to implement the table information access interface.